packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py

```diff
@@ -0,0 +1,8 @@
+import re
+
+from models_library.basic_regex import PROPERTY_KEY_RE
+from pydantic import ConstrainedStr
+
+
+class PortKey(ConstrainedStr):
+    regex = re.compile(PROPERTY_KEY_RE)
```

Comment:
What is the difference with ```ServicePortKey``` in models_library.services_types.py ?

Reply:
None apparently. Let's try to reuse it. I will remove this one.

Reply:
@sanderegg I have also removed it from the dask-task-models-library